Dockerfile.frontend

```diff
@@ -0,0 +1,37 @@
+FROM node:10.12.0-alpine as builder
+
+WORKDIR /app
+
+# minimize included packages
+RUN mkdir packages
+COPY ./packages/code-du-travail-frontend ./packages/code-du-travail-frontend
+COPY ./packages/code-du-travail-css ./packages/code-du-travail-css
+COPY ./packages/code-du-travail-ui ./packages/code-du-travail-ui
+
+RUN mkdir scripts
+COPY ./scripts/setup-env.sh ./scripts/setup-env.sh
+
+COPY package.json .
+COPY yarn.lock .
+COPY lerna.json .
+
+RUN yarn
+
+# Variables needed at build time for the frontend...
+# TODO: we should switch to https://github.com/zeit/next.js/tree/master/examples/with-universal-configuration-runtime ?
+ARG API_URL
+ARG INTERNAL_API_URL
+ARG SENTRY_PUBLIC_DSN
+ARG PIWIK_URL
+ARG PIWIK_SITE_ID
+
+ENV NODE_ENV=production
+ENV API_URL=$API_URL
+ENV INTERNAL_API_URL=$INTERNAL_API_URL
```

Comment:
what 's the difference between `API_URL` and `INTERNAL_API_URL` ?

Reply:
yep INTERNAL will not exist but let me explain where it came from as i wasted waaay too much time on this :)

We're currently using next.js which does SSR. 

When the user request some content client-side, like autosuggest, he uses his browser `fetch()` and the public `process.env.API_URL` to resolve the data from our API. all fine.

But when the user request a *page* of content like `/question/how-are-you-today` then that's the server who make the initial request to the API and inject content into the page before returning it to user. this happens in the pages `getInitialProps()` function, which also use `process.env.API_URL`.

So at some point, the server use the public API_URL to make requests to the API server which in fact stands on the same docker-compose environment. 

https://github.com/zeit/next.js/#fetching-data-and-component-lifecycle

Reply:
Also these variables are used at build time and i think we should make them overridable at runtime